src/airtable/proposals/proposal_standings.js

```diff
@@ -185,13 +188,19 @@ const getProposalRecord = async (proposal, allProposals, curRoundNumber) => {
   const proposalURL = proposal.get('Proposal URL')
 
   let lastOceanBalanceCheckDate = proposal.get('Last Balance Check')
-  let areOceansEnough = proposal.get('Proposal Standing') !== 'No Ocean'
+  let areOceansEnough =
+    proposal.get('Proposal Standing') !== '' &&
+    proposal.get('Proposal Standing') !== 'No Ocean'
   if (
-    !(curRoundNumber !== undefined && curRoundNumber !== proposal.get('Round'))
+    !(
+      curRoundNumber !== undefined &&
+      parseInt(curRoundNumber) !== parseInt(proposal.get('Round'))
+    )
   ) {
     if (
       !lastOceanBalanceCheckDate ||
-      new Date(lastOceanBalanceCheckDate) + 1000 * 60 * 15 < new Date() // undefined or 15 minutes has passed
+      new Date(lastOceanBalanceCheckDate).getTime() + 1000 * 60 * 15 <
+        Date.now() // undefined or 15 minutes has passed
```

Comment:
Which linter is recommending this change?
Does this break `npm run lint`?

Reply:
Yes